spec/models/pixelz/modification_request_spec.rb

```diff
@@ -46,4 +46,25 @@
       end
     end
   end
+
+  describe '#reject' do
+    before do
+      @image = create(:image)
+      @modify_image_request = Pixelz::ModificationRequest.modify(@image) 
```

Comment:
We should use a factory-created `Pixelz::ModificationRequest` in this type of isolated test so that failures in the `::modify` method do not bleed into the `#reject` specs